### ISSUE TITLE
Support FFmpeg 6.0

### ISSUE THREAD

**Issue by AvrahamTsaban** [audacity/audacity#4480]
FFmpeg 6.0 (avformat-60.dll) is released.
 According to FFmpeg [news](https://ffmpeg.org/index.html#news), it seems to be less consistent withe previous versions than prefious minor updates. 
I assume audacity 3.3 should be compatible with it. 
I use it to decode files on Audacity 3.2.5 and it works well, for now.

**Comment by dozzzzer** [comment]
Duplicate of #4381

**Comment by SakiPapa** [comment]
Audacity for macOS does not recognize ffmpeg 6.0 libraries

I am using Audacity 3.2.5 on macOS. (M1 PowerBook 13 Pro, Monterey 12.6.5, 16 GB memory)

My ffmpeg library (Arm version) was installed by Homebrew.

Yesterday after I updated ffmpeg to version 5.1.2 version 6.0, I got a warning that the library could not be found.

I confirmed that the ffmpeg libraries are installed in

/opt/homebrew/Cellar/ffmpeg/6.0/.

I restored the old ffmpeg (5.1.2_6) from the TimeMachine backup back to

/opt/homebrew/Cellar/ffmpeg/5.1.2_6

and Audacity recognized the 5.1.2 library.

I think the macOS version of Audacity 3.2.5 (Apple Silicon) is not recognize ffmpeg 6.0.

The same result was occured with Audacity 3.3.0-beta1.
